src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/ApplicationTimelineTransformerTest.kt

```diff
@@ -106,6 +110,44 @@ class ApplicationTimelineTransformerTest {
     )
   }
 
+  @ParameterizedTest
+  @MethodSource("domainEventTypeArgs")
+  fun `transformDomainEventSummaryToTimelineEvent adds appealUrl for appeal events and the user can manage appeals`(args: Pair<DomainEventType, TimelineEventType>) {
```

Comment:
I don't think this bit is relevant? The link to view the appeal page should be present for all users I believe?

>  and the user can manage appeals

Reply:
Good catch!